cmake/kokkos_arch.cmake

```diff
@@ -573,7 +573,7 @@ FUNCTION(CHECK_CUDA_ARCH ARCH FLAG)
     ENDIF()
     SET(CUDA_ARCH_ALREADY_SPECIFIED ${ARCH} PARENT_SCOPE)
     IF (NOT KOKKOS_ENABLE_CUDA AND NOT KOKKOS_ENABLE_OPENMPTARGET AND NOT KOKKOS_ENABLE_SYCL AND NOT KOKKOS_ENABLE_OPENACC)
-      MESSAGE(WARNING "Given CUDA arch ${ARCH}, but Kokkos_ENABLE_CUDA, Kokkos_ENABLE_OPENACC, and Kokkos_ENABLE_OPENMPTARGET are OFF. Option will be ignored.")
+      MESSAGE(WARNING "Given CUDA arch ${ARCH}, but Kokkos_ENABLE_CUDA, Kokkos_ENABLE_SYCL, Kokkos_ENABLE_OPENACC, and Kokkos_ENABLE_OPENMPTARGET are OFF. Option will be ignored.")
```

Comment:
I think we left this off here intentionally since its unsupported.

Reply:
@masterleinad thoughts?

Reply:
On the other hand we print them above, so I guess could go either way.

Reply:
I think it makes sense to print `SYCL` in the warning as well.